cmd/lit-af/wsRPC.go

```diff
@@ -0,0 +1,115 @@
+package main
```

Comment:
a large part of this was taken from https://golang.org/src/net/rpc/jsonrpc/client.go but I'm not sure how to properly cite it